src/library/vtkF3DLog.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * @class   vtkF3DLog
+ * @brief   Class used to show logs
+ *
+ */
+
+#ifndef vtkF3DLog_h
+#define vtkF3DLog_h
+
+#include <vtkObject.h>
+
+#include <sstream>
+#include <string>
+
+// TODO could this be a namespace instead of inheriting vtkObject ?
```

Comment:
Yes, I don't see the point of inheriting from vtkObject here, especially if we never instantiate the object.

Reply:
I was woried that iy may not have worked with vtk modules for some reason, but it seems to work fine. It may not be python wrappable but we do not care about that.